src/main/java/com/kodilla/ecommercee/exception/GlobalHttpErrorHandler.java

```diff
@@ -22,4 +22,20 @@ public ResponseEntity<Object> handlerRecordNoExistsException(RecordNotExistsExce
     public ResponseEntity<Object> handlerRecordExistsException(RecordExistsException excetpion) {
         return new ResponseEntity<>("Object already exist", HttpStatus.BAD_REQUEST);
     }
-}
\ No newline at end of file
+
+    @ExceptionHandler(OrderNotFoundException.class)
+    public ResponseEntity<Object> handleOrderNotFoundException(OrderNotFoundException exception) {
+        return new ResponseEntity<>("One or more of required fields is incorrect", HttpStatus.BAD_REQUEST);
```

Comment:
Here should be "Order with given id doesn't exist."

Reply:
Ok, I've changed it